sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_version.py

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict, Union
+from azure.ai.ml._schema._deployment.batch.batch_job_property import OutputDataSchema
+from azure.ai.ml._schema.core.fields import ArmStr
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+
+
+class DataVersion:
+    """Batch Job Resource entity
```

Comment:
Please update the documentation properly. Seems that this was for Batch Job Resource not DataVersion